wgpu-types/src/lib.rs

```diff
@@ -764,6 +764,22 @@ impl BlendComponent {
         operation: BlendOperation::Add,
     };
 
+    /// Color blend mode that does standard alpha blending with non-premultiplied alpha.
+    /// The alpha blend mode often used is REPLACE.
+    pub const COLOR_BLEND: Self = BlendComponent {
```

Comment:
Isn't this generally called "alpha blending"? I.e. maybe I'm thinking about this the wrong way, but the model I have in the head is that "blending" term by itself assumes the color, and "alpha blending" means mix colors based on the alpha. From that perspective, "color blend" does not make much sense.

Reply:
I'm just worried that people will put this in the alpha with REPLACE in color if it's called alpha. Idk if there's a pretty solution to this.

Reply:
ah I see. I think one solution could be to define these constants on `impl BlendState` instead of the blend component. Did you consider this? It would be even more ergonomic this way.

Reply:
Alternative proposal: `ALPHA_BLEND_COLOR_COMPONENT` and `PREMULTIPLIED_ALPHA_BLEND_COLOR_COMPONENT`, which I think is a bit more explicit.

Reply:
do you see the need for this (versus just `BlendState` constants)?

Reply:
I'd be happy with blend state constants, which would eliminate this issue entirely.

Reply:
let's just do it then